context/app/static/js/components/workspaces/AdvancedConfigOptions/AdvancedConfigOptions.tsx

```diff
@@ -45,37 +45,30 @@ function ConfigSlider<FormType extends FieldValues>({
   conversionFactor = 1,
   markInterval = 1,
 }: ConfigSliderProps<FormType>) {
-  const convert = (value: number) => value / conversionFactor;
-  const unconvert = (value: number) => value * conversionFactor;
+  const convertedMin = convert(min, conversionFactor);
+  const convertedMax = convert(max, conversionFactor);
 
-  const convertedMin = convert(min);
-  const convertedMax = convert(max);
+  // Show even numbers if the interval is even, otherwise show odd numbers
+  const start = markInterval % 2 === 0 ? convertedMin + 1 : convertedMin;
 
-  const marks = useMemo(() => {
-    const tempMarks = [];
-    // Show even numbers if the interval is even, otherwise show odd numbers
-    const start = markInterval % 2 === 0 ? convertedMin + 1 : convertedMin;
-
-    for (let i = start; i <= convertedMax; i += markInterval) {
-      tempMarks.push({ value: i, label: i });
-    }
-
-    return tempMarks;
-  }, [convertedMin, convertedMax, markInterval]);
+  const marks = Array.from({ length: Math.floor((convertedMax - start) / markInterval) + 1 }, (_, i) => {
+    const value = start + i * markInterval;
+    return { value, label: value };
+  });
 
   return (
     <Stack marginTop={1}>
       <Stack direction="row" spacing={1} alignItems="center">
-        <StyledSubtitle>{label}</StyledSubtitle>
+        <Typography variant="button">{label}</Typography>
```

Comment:
Button without an `onClick`?

Reply:
This was the variant that matched the slider labels in the designs - would custom styling be preferable in this case ?

(with button variant)
![Screenshot 2024-09-05 at 2 23 42 PM](https://github.com/user-attachments/assets/a1ffe4ff-084b-48ab-9c5b-c92f957f078e)
(without variant)
![Screenshot 2024-09-05 at 2 23 28 PM](https://github.com/user-attachments/assets/f1ec8304-c0c9-4fa5-bd39-2128ad29c5b5)

Reply:
The closest other variant is `subtitle2` - however, this variant is colored `primary` instead of black. Would it be preferable to use this and alter the font color?

Reply:
Yes or introduce a new typography variant to the theme. The button variant comes with other accessibility changes and impacts readability.

Reply:
That typography should be `subtitle2`, but the black color. I might have messed up the color for the mockup.

Reply:
Ok great, thanks for clarifying! I'll use `subtitle2` with a black font color.